src/components/templates/Jazzbar/components/JazzBarRoom/Room.tsx

```diff
@@ -0,0 +1,322 @@
+import React, {
```

Comment:
This file seems to duplicate our generic component `src/components/organisms/Room/Room.tsx`

What's the reasoning/need for duplicating it?

Reply:
The only differences appear to be:

```diff
diff --git a/src/components/organisms/Room/Room.tsx b/src/components/templates/Jazzbar/components/JazzBarRoom/Room.tsx
index 41bb9832..7e3174dd 100644
--- a/src/components/organisms/Room/Room.tsx
+++ b/src/components/templates/Jazzbar/components/JazzBarRoom/Room.tsx
@@ -222,7 +222,9 @@ const Room: React.FC<RoomProps> = ({
 
   const meComponent = useMemo(() => {
     return room && profileData ? (
-      <div className={`participant-container ${participantContainerClassName}`}>
+      <div
+        className={`jazzbar-participant-container ${participantContainerClassName}`}
+      >
         <LocalParticipant
           key={room.localParticipant.sid}
           participant={room.localParticipant}
@@ -255,7 +257,7 @@ const Room: React.FC<RoomProps> = ({
         return (
           <div
             key={participant.identity}
-            className={`participant-container ${participantContainerClassName}`}
+            className={`jazzbar-participant-container ${participantContainerClassName}`}
           >
             <Participant
               key={`${participant.sid}-${index}`}
@@ -282,7 +284,7 @@ const Room: React.FC<RoomProps> = ({
         ? Array(participants.length % 2).map((e, index) => (
             <div
               key={`empty-participant-${index}`}
-              className={`participant-container ${participantContainerClassName}`}
+              className={`jazzbar-participant-container ${participantContainerClassName}`}
             >
               <img
                 className="empty-chair-image"
@@ -295,6 +297,8 @@ const Room: React.FC<RoomProps> = ({
     [hasChairs, participants.length, participantContainerClassName]
   );
 
+  console.log({ emptyComponents, hasChairs });
+
   if (!token) {
     return <></>;
   }
```